packages/editor/src/store/private-selectors.js

```diff
@@ -120,29 +118,6 @@ export const getPostIcon = createRegistrySelector(
 	}
 );
 
-/**
- * Returns the template parts and their blocks for the current edited template.
- *
- * @param {Object} state Global application state.
- * @return {Array} Template parts and their blocks in an array.
- */
-export const getCurrentTemplateTemplateParts = createRegistrySelector(
```

Comment:
This selector is still "aliased" in the site editor store and triggers an error now.

Should we also remove it?

https://github.com/WordPress/gutenberg/blob/387acefdeb14d4d8aeb2885b840c2ffa50562b64/packages/edit-site/src/store/selectors.js#L241-L247